label_sync/main.go

```diff
@@ -217,7 +217,34 @@ func stringInSortedSlice(a string, list []string) bool {
 	return false
 }
 
-// Ensures the config does not duplicate label names
+// Returns a sorted list of labels unique by name
+func (c Configuration) Labels() []Label {
+	var labelarrays [][]Label
+	labelarrays = append(labelarrays, c.Default.Labels)
+	for _, repo := range c.Repos {
+		labelarrays = append(labelarrays, repo.Labels)
+	}
+
+	labelmap := make(map[string]Label)
+	for _, labels := range labelarrays {
+		for _, l := range labels {
+			name := strings.ToLower(l.Name)
```

Comment:
Seems like we should invalidate labels that are case-sensitive permutations of each other -- that's most likely a user error in `labels.yaml`, right?